share/arno-iptables-firewall/aif-job-execute

```diff
@@ -101,7 +101,9 @@ fi
 
 # Check if the environment file exists and if so, load it
 #########################################################
-if [ -n "$ENV_FILE" ]; then
+ENV_FILE="$(dirname $0)/environment"
+
+if [ -e "$ENV_FILE" ]; then
```

Comment:
Another `-f` test is better

Reply:
Well, doesn't -f skip symlinks?

Reply:
Just checked: you're right, -f also works for symlinks.

Reply:
No, `-f` follows symlinks.